lmdeploy/pytorch_poc/engine/cache_engine.py

```diff
@@ -29,6 +29,7 @@ def __init__(
         self,
         cache_config: CacheConfig,
         model_config: ModelConfig,
+        json_config: dict,
```

Comment:
Since json_config is basically huggingface config, I think it would be better if we can merge it into model_config(which is designed to be an adapter of huggingface config)

Reply:
done.